src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextHostInstanceTests.cs

```diff
@@ -255,12 +255,12 @@ private WorkspaceProjectContextHostInstance CreateInstance(ConfiguredProject pro
             applyChangesToWorkspaceContext = applyChangesToWorkspaceContext ?? IApplyChangesToWorkspaceContextFactory.Create();
 
             return new WorkspaceProjectContextHostInstance(project,
-                                                    threadingService,
-                                                    tasksService,
-                                                    projectSubscriptionService,
-                                                    workspaceProjectContextProvider,
-                                                    activeWorkspaceProjectContextTracker,
-                                                    ExportFactoryFactory.ImplementCreateValueWithAutoDispose(() => applyChangesToWorkspaceContext));
+                                                           threadingService,
```

Comment:
This is why I prefer to put the first parameter on the next line indented by 4 when wrapping parameters.  That way it never has to move if the method is renamed.

Reply:
The diff still looks rather reasonable with `?w=1` turned on.

Reply:
Yeah, but history/annotate view is littered with people adjusting whitespace. ?w=1 is a bandaid.

Reply:
Do you suggest that we put the first parameter on a newline? Or just the second parameter and later?

Reply:
I interpreted @Pilchie's comment as meaning if you wrap, there's a newline before the first parameter. Likewise for argument lists. I might be biased because that's my preferred way of formatting such lists (also to minimise white space churn and maintenance).

It's also consistent when nesting calls (in rare cases where that's clearer than using temporaries) such as:

```c#
Method1(
    arg1,
    Method2(
        arg2,
        arg3),
    arg4);
```

Reply:
What @drewnoakes said.  That way if the method is renamed, qualified with this, etc, the indentation of all the parameter/argument lines doesn't need to be adjusted.

Reply:
Yep sounds good. Roslyn has a refactoring that does this, so its even easier.